crates/matrix-sdk-common/src/executor.rs

```diff
@@ -47,24 +73,28 @@ where
         let _ = future.await;
     });
 
-    JoinHandle { remote_handle: Some(remote_handle), abort_handle }
+    JoinHandle { remote_handle: Some(remote_handle), the_abort_handle: abort_handle }
 }
 
 #[cfg(target_arch = "wasm32")]
 #[derive(Debug)]
 pub struct JoinHandle<T> {
     remote_handle: Option<RemoteHandle<T>>,
-    abort_handle: AbortHandle,
+    the_abort_handle: AbortHandle,
```

Comment:
Why rename this?

Reply:
That's not true, you can have a method with the same name as a field just fine. It's somewhat common, even.